lib/virt_autotest/utils.pm

```diff
@@ -379,7 +379,7 @@ sub ensure_online {
         }
         unless ($skip_ssh == 1) {
             # Wait for ssh to come up
-            die "$guest does not start ssh" if (script_retry("nmap $guest -PN -p ssh | grep open", delay => 15, retry => 12) != 0);
+            die "$guest does not start ssh" if (script_retry("nmap $guest -PN -p ssh | grep open", delay => 30, retry => 12, timeout => 360) != 0);
```

Comment:
This will increase the timeout for all operations calling this function and make test fail slower overall, right? If you find this delay mainly come for specific guest, would you please only extend the timeout for it ?

Reply:
Actually, current TIMEOUT for our OSD test run due to 15-SP5 PV guest take a long time within 6 mins to finish reboot(i.e. bsc#1206250).  Depend on current situation, this func ensure_online just only deal with this 15-SP5 PV guest reboot problem to ensure given guest system is ready for our feature tests and continue our following OSD test module. 

yes, good point, add the only extend the timeout for the specific guest - 15-SP5 PV guest system now.

Reply:
This is not updated to be for 15-SP5 PV guest  yet ?